### PR TITLE
Fix lint in smallbank workload

### PR DESCRIPTION
**Pull request by boydjohnson**

### CODE REVIEW

File: perf/smallbank_workload/src/smallbank_transformer.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ pub struct SBPayloadTransformer<'a> {
 impl<'a> SBPayloadTransformer<'a> {
     pub fn new(signer: &'a signing::Signer) -> Self {
         SBPayloadTransformer {
-            signer: signer,
+            signer,
```
Comment (aludvik) [inline perf/smallbank_workload/src/smallbank_transformer.rs]: This should be a separate commit

Comment (boydjohnson) [reply perf/smallbank_workload/src/smallbank_transformer.rs]: Thanks. Will fix that.
